gradle.properies

```diff
@@ -0,0 +1,5 @@
+org.gradle.jvmargs=-Xmx1024 -XX:MaxPermSize=256 -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
```

Comment:
Is 1GB an appropriate max for the compiler? Especially if compiling in parallel? Is this easily configurable so I can let a beefy build-server max out?